test/unit/reports/get-currency.test.js

```diff
@@ -0,0 +1,25 @@
+const { UNKNOWN } = require('../../../app/constants/unknown')
+
+const extractedEvent = require('../../mocks/events/extracted')
+const enrichedEvent = require('../../mocks/events/enriched')
+
+const { getCurrency } = require('../../../app/reports/mi-report/get-currency')
```

Comment:
Maybe group 'UNKOWN' and 'getCurrency'?

Reply:
I kept `getCurrency` separate as it's the focus of the tests so wanted it to stand out